data/application/ohmyclass-init.sql

```diff
@@ -0,0 +1 @@
+CREATE DATABASE IF NOT EXISTS `ohmyclass` DEFAULT CHARACTER SET utf8 COLLATE utf8_general_ci;
```

Comment:
Do we even need this, if we have the **MYSQL_DATABASE** var on line 14 in the docker-compose.yml?